media/windows/calendar-reservations/calendar-reservations.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+#
+# You need all the Python packages in requirements.txt.
```

Comment:
I forgot to include this in my review: this comment references the file `requirements.txt`, but that's not included in this PR.  You should probably add it (which will then make this comment be valid).

I think the contents of `requirements.txt` for this project needs to be:

```
httplib2
apiclient
google-api-python-client
oauth2client
```

These are the 3rd-party Python modules that you need to have in your environment to make `calendar-reservations.py` work properly.